docs/Deploying/Configuration.md

```diff
@@ -2249,99 +2249,18 @@ Fleet supports the SAML Web Browser SSO Profile using the HTTP Redirect Binding.
 
 _**Note: The email used in the SAML Assertion must match a user that already exists in Fleet unless you enable [JIT provisioning](#just-in-time-jit-user-provisioning).**_
 
-##### sso_settings.enable_sso
-
-Configures if single sign-on is enabled.
-
-- default value: false
-- config file format:
-  ```
-  sso_settings:
-    enable_sso: true
-  ```
-
-##### sso_settings.enable_sso_idp_login
-
-Allow single sign-on login initiated by identity provider.
-
-- default value: false
-- config file format:
-  ```
-  sso_settings:
-    enable_sso_idp_login: true
-  ```
-
-##### sso_settings.enable_jit_provisioning
-
-Enables [just-in-time user provisioning](#just-in-time-jit-user-provisioning)
-
-- default value: false
-- config file format:
-  ```
-  sso_settings:
-    enable_jit_provisioning: true
-  ```
-
-##### sso_settings.entity_id
-
-The required entity ID is a URI that you use to identify Fleet when configuring the identity provider.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    entity_id: "https://example.com"
-
-##### sso_settings.idp_image_url
-
-An optional link to an image such as a logo for the identity provider.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    idp_image_url: "https://example.com/logo"
-
-##### sso_settings.idp_name
-
-A required human friendly name for the identity provider that will provide single sign-on authentication.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    idp_name: "SimpleSAML"
-
-##### sso_settings.issuer_uri
-
-The issuer URI supplied by the identity provider.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    issuer_uri: "https://example.com/saml2/sso-service"
-
-##### sso_settings.metadata
-
-Metadata provided by the identity provider. Either metadata or a metadata url must be provided.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    metadata: ""
-
-##### sso_settings.metadata_url
-
-A URL that references the identity provider metadata.
-
-- default value: ""
-- config file format:
-  ```
-  sso_settings:
-    metadata: "https://example.com/saml2/metadata"
+<!-- Some subsections were renamed, but heading are kept so that link fragments still work -->
+<span id="sso-settings-enable-sso" name="sso-settings-enable-sso"></span>
+<span id="sso-settings-enable-sso-idp-login" name="sso-settings-enable-sso-idp-login"></span>
+<span id="sso-settings-enable-jit-provisioning" name="sso-settings-enable-jit-provisioning"></span>
+<span id="sso-settings-entity-id" name="sso-settings-entity-id"></span>
+<span id="sso-settings-idp-image-url" name="sso-settings-idp-image-url"></span>
+<span id="sso-settings-idp-name" name="sso-settings-idp-name"></span>
+<span id="sso-settings-issuer-uri" name="sso-settings-issuer-uri"></span>
+<span id="sso-settings-metadata" name="sso-settings-metadata"></span>
+<span id="sso-settings-metadata-url" name="sso-settings-metadata-url"></span>
```

Comment:
The idea was more in case some of those link fragments were saved/bookmarked/referenced elsewhere from users, third-party blog posts, internal company documentation, etc. so that we don't break any links. I'm fine with removing them if we think that's being overly cautious.

> I like the thinking/technique here. Are we using this <span /> technique in other places in the docs?

Yes, AFAIK @roperzh came up with them to avoid breaking links when we rename section headers (at least that's on one of his PRs that I noticed them for the first time).

Reply:
+1 that particular PR had a link from within the Fleet UI, but if I might chime in I still think it's a very low effort way of covering the cases Martin mentioned, plus many more, eg: links we shared in community channels, search engines, etc.

Reply:
> in case some of those link fragments were saved/bookmarked/referenced elsewhere from users, third-party blog posts, internal company documentation, etc.

> low effort way of covering links we shared in community channels, search engines, etc.

Really good points.

I think the headings in the docs will experience a lot of change in the near future. I think this means that if we add old link fragments (`<span>` elements) every time time we rename a heading, the list of old link fragments will become hard to maintain. 

I think we want to balance the usability of the docs with maintainability. My suspicion is that the value we get from maintaining links for third-party blog posts, community channels, and SEO is relatively low.

As a result, my proposal is to only add old link fragments when renaming a heading breaks links in the Fleet UI or fleetctl. We can always update the docs and Fleet articles to the latest link in the same pass.

What do you think?

Looping in folks from the digital experience team and folks with SEO experience for thoughts: @mike-j-thomas @mikermcneil

Reply:
Given that those are only the fragment part of the links (the `#something`), they probably have a very low importance in SEO (if at all), but that's really not an area I know a lot about.

> I think this means that if we add old link fragments (<span> elements) every time time we rename a heading, the list of old link fragments will become hard to maintain.

I agree. I'll wait for comments on the effect on SEO, but if it's minimal/not worth the trouble, I'll remove those backlinks.

Reply:
I'd focus on usability over SEO.  The best SEO is to make an excellent doc page (on its own URL) with good, accurate headings and a nice, clean URL.  It should be easy to read and match our styleguide.

If unsure, get help from @chris-mcgillicuddy, who is the DRI for aesthetics and style guide in the docs.

Thanks @mna and @roperzh!  So excited for this.  The work on improving docs is one of the highest leverage contributions any of us can make right now.